platform/linuxbsd/display_server_x11.cpp

```diff
@@ -3095,7 +3178,7 @@ DisplayServerX11::WindowID DisplayServerX11::_create_window(WindowMode p_mode, c
 		wd.size.width = xwa.width;
 		wd.size.height = xwa.height;
 
-		//print_line("created at rect: " + p_rect + " but at rect " + Rect2i(xwa.x, xwa.y, xwa.width, xwa.height));
+		print_line("created at rect: " + p_rect + " but at rect " + Rect2i(xwa.x, xwa.y, xwa.width, xwa.height));
```

Comment:
Should this stay enabled, or was it only for temporary debugging?

Reply:
Is very useful, I use this for the knowing bugs.